.dockerignore

```diff
@@ -1,2 +1,37 @@
 .env*
-.gitignore
\ No newline at end of file
+.gitignore
+
+# Local .terraform directories
+**/.terraform/*
```

Comment:
This looks like you've taken a standard `.gitignore` and also applied it to your `.dockerignore` which is a very sensible policy but it's worth being aware that despite the similarity there are some syntax differences between them which can catch you out - [see e.g. this article](https://zzz.buzz/2018/05/23/differences-of-rules-between-gitignore-and-dockerignore/) for examples. I _believe_ they'll both work with the `**/<directory/*` syntax but I'm not certain of that, the way it catches me out is the inverse where `.gitignore` would also accept `<directory>` but `.dockerignore` won't. Nothing to change if it's working, but thought worth mentioning!